Updates to Docker Scout overview and advanced image analysis pages (#16842)
docker/docs: scout/advanced-image-analysis.md

> **Note**
>
> Docker Scout is an [early access](../release-lifecycle.md#early-access-ea)

> > product, and requires a Docker Pro, Team, or Business subscription..

Reviewer: ```suggestion
> product, and requires a Docker Pro, Team, or Business subscription.
```

>

> > If you're interested in this integration for your organization and want to

Reviewer: ```suggestion
> If you're interested in Docker Scout for your organization and want to
```

> learn more, get in touch by filling out the contact form on the
> [Docker Scout product page](https://docker.com/products/docker-scout){:
> target="\_blank" rel="noopener" }.

Advanced image analysis is a Docker Scout feature for Docker Hub.

> When you activate advanced image analysis for a repository, Scout analyzes new tags

Reviewer 2: I believe having 'advanced' capitalized was correct. It's also now inconsistent here with the rest of the page.

Author: ```suggestion
When you activate Advanced image analysis for a repository, Scout analyzes new tags
```

To view the vulnerability report on Docker Hub:

1. Go to Docker Hub and open the repository page. The **Tags** section
   displays a vulnerability summary.

   It may take a few minutes for the vulnerability report to appear. If your vulnerability summary doesn't display, wait a moment
   and then refresh the page.

2. Click on the tag in the table. This opens the details page for the tag.

3. Select the **Vulnerabilities** tab on the right side of the page.

   This tab displays a deep-dive view of the image's packages and any known vulnerabilities.

   For more information about how to interpret the vulnerability report, see
   [Image details view](./image-details-view.md).

Expanding any of the packages in the list shows you more information about the
vulnerabilities that affect a given package. Expanding the vulnerability shows a summary of it's details and

> clicking on the vulnerability name opens Docker's image vulnerability database, which provides

Reviewer 2: ```suggestion
selecting the vulnerability name opens Docker's image vulnerability database, which provides
```
We try to avoid using 'click' https://docs.docker.com/contribute/style/recommended-words/

Author: @aevesdocker Yes indeed, I saw that in VSCode too. For some reason when I read it, it didn't sound right in this context, but now it does, so 🤷‍♂️
